src/plugins/intel_gpu/src/graph/mtcnn-o2.xml

```diff
@@ -0,0 +1,1685 @@
+<?xml version="1.0" ?>
+<net name="ONet" version="11">
```

Comment:
Can we avoid physical files with models? Can it be a part of test body?

Reply:
Sorry, I added this by mistake. Removed, thanks